app.js

```diff
@@ -198,4 +198,34 @@ modalBtn.forEach((el, index) => {
     const closeModalBtn = document.querySelector('.lass');
     closeModalBtn.addEventListener('click', closeModal);
   });
-});
\ No newline at end of file
+});
+
+const form = document.querySelector('#form');
+const emailInput = document.querySelector('#email');
+const errorMsgContainer = document.querySelector('#errorMsg');
+const errorMsg = '* Hi there! Please, Check it out again pls, thanks.*';
```

Comment:
- [ ] You did a great job adding the error message but please change the design that you gave to it because it taking to much space and it's passing the ending of the portfolio. For example, you can decrease a little teh font size. Keep coding 💪 

Here you have a screenshot of the error message from your project:

![image](https://user-images.githubusercontent.com/74506933/162592854-9397c138-a98b-4277-8ea1-6fcadaa4bcb4.png)

[OPTIONAL] Kindly change the content of the button because it is Get in touch and not Get my resume. I suggest you to do it after the error message design even if this request doesn't stop you from getting approved. Keep shining 💪